src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/CarrierType.php

```diff
@@ -49,12 +49,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
         $builder
+            ->add('shipping_settings', ShippingLocationsAndCostsType::class, [
```

Comment:
I thought the shipping costs were supposed to be the second tab no? Maybe you moved it for faster development?